Realm/RLMSyncSessionRefreshHandle.mm

```diff
@@ -270,12 +273,11 @@ - (void)_timerFired {
     RLMSyncCompletionBlock handler = ^(NSError *error, NSDictionary *json) {
         [weakSelf _onRefreshCompletionWithError:error json:json];
     };
-// FIXME: [realmapp] This still potentially has use. It is unclear at the moment
-// FIXME: if this should be handled in the OS or at this layer.
-//    [RLMSyncAuthEndpoint sendRequestToServer:self.authServerURL
-//                                        JSON:json
-//                                     timeout:60.0
-//                                  completion:handler];
+    [RLMSyncAuthEndpoint sendRequestToServer:self.authServerURL
+                                        JSON:json
+                                     timeout:60.0
+                                  completion:handler];
+#endif
```

Comment:
I have investigated this, there are several tasks that need to be completed first before this is fixed.

Reply:
This change is just shuffling around how much is commented out to eliminate the unused variable warnings.